### PR TITLE
Fixed #26899 -- Document why RawSQL params is an explicit parameter

### PR DESCRIPTION
**Pull request by petedmarsh**
A small change for: https://code.djangoproject.com/ticket/26899. Happy to change/amend as you see fit! :)

### CODE REVIEW

**Comment by timgraham** [comment]
buildbot, add to whitelist.

**Comment by petedmarsh** [comment]
As discussed in the ticket I've changed this to explain why `params` is an explicit parameter rather than making it an optional parameter.

File: docs/ref/models/expressions.txt [hunk 1]
```diff
@@ -459,7 +459,8 @@ should avoid them if possible.
 
     You should be very careful to escape any parameters that the user can
     control by using ``params`` in order to protect against :ref:`SQL injection
-    attacks <sql-injection-protection>`.
+    attacks <sql-injection-protection>`. ``params`` is an explicit argument to
```
Comment (timgraham) [inline docs/ref/models/expressions.txt]: How about: ``params`` is a required argument to force you to acknowledge that you're not interpolating data in your SQL.

**Comment by petedmarsh** [comment]
@timgraham I went for something similar to what you suggested, just clarified that it's user-provided data that you need to be careful with

**Pull request merged**